### PR TITLE
{lib}[foss/2022a] TensorFlow v2.10.1 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

I'm working on TF 2.11.0 but that is causing me some trouble, so adding this as an intermediate in addition to 2.9.1

Edit: I think I got TF 2.11.0 working and will open a PR at start of February as it needs polishing. So if that is short enough to skip TF 2.10.1 then this can be closed.

Edit2: TF 2.11.0 PR is at https://github.com/easybuilders/easybuild-easyconfigs/pull/17241

### CODE REVIEW

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in total)
taurusi8028 - Linux CentOS Linux 7.9.2009, x86_64, AMD EPYC 7352 24-Core Processor (zen2), 8 x NVIDIA NVIDIA A100-SXM4-40GB, 470.57.02, Python 2.7.5
See https://gist.github.com/212c88f429c4ff50eecf7fbc5fbfc7cd for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
taurusml20 - Linux RHEL 7.6, POWER, 8335-GTX (power9le), 6 x NVIDIA Tesla V100-SXM2-32GB, 440.64.00, Python 2.7.5
See https://gist.github.com/6e3205a258932d259e9b9b7768e34df1 for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
taurusa12 - Linux CentOS Linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2603 v4 @ 1.70GHz (broadwell), 3 x NVIDIA GeForce GTX 1080 Ti, 460.32.03, Python 2.7.5
See https://gist.github.com/64fa1eb28489f488c5402fb393ca3e1b for a full test report.

**Comment by branfosj** [comment]
I think the CUDA version will require the changes in https://github.com/easybuilders/easybuild-easyblocks/pull/2854

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 5 out of 6 (3 easyconfigs in total)
node406.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, AMD EPYC 7282 16-Core Processor, 1 x NVIDIA NVIDIA A100-PCIE-40GB, 515.48.07, Python 3.6.8
See https://gist.github.com/172b9493193381a875dfa95ac919077a for a full test report.

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.10.1-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('Bazel', '5.1.1'),
+    ('protobuf', '3.19.4'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.36.0', '-nodocs'),
+    ('pybind11', '2.9.2'),
+    ('UnZip', '6.0'),
+    ('LLVM', '14.0.3'),  # for debugging with llvm-symbolizer, to be removed
```
Comment (branfosj) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.10.1-foss-2022a-CUDA-11.7.0.eb:20]:
```suggestion
```

Removing the debug `LLVM` build dep.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/4ac9519ae5c8bad6e1ad707a322212b3 for a full test report.

**Comment by Flamefire** [comment]
@branfosj As we have TF 2.11 and 2.9 do we still want this? I'd like to just skip 2.10 as I don't see any benefit in spending more time on this. However it would be important that #17092 gets merged, so we actually have the fixed 2.9.1

**Comment by casparvl** [comment]
Agreed, I'll close this.

**Pull request closed without merging**